docs/tutorials/unsupervised_learning/gan.md

```diff
@@ -86,25 +86,25 @@ Since the DCGAN that we're creating takes in a 64x64 image as the input, we'll u
 import cv2
 X = np.asarray([cv2.resize(x, (64,64)) for x in X])
 ```
-Each pixel in our 64x64 image is represented by a number between 0-255, that represents the intensity of the pixel. However, we want to input numbers between -1 and 1 into our DCGAN, as suggested by the research paper. To rescale our pixels to be in the range of -1 to 1, we'll divide each pixel by (255/2). This put our images on a scale of 0-2. We can then subtract by 1, to get them in the range of -1 to 1.
+Each pixel in the 64x64 image is represented by a number between 0-255, that represents the intensity of the pixel. However, we want to input numbers between -1 and 1 into the DCGAN, as suggested by the research paper. To rescale the pixels to be in the range of -1 to 1, we'll divide each pixel by (255/2). This put the images on a scale of 0-2. We can then subtract by 1, to get them in the range of -1 to 1.
 ```python
 X = X.astype(np.float32)/(255.0/2) - 1.0
 ```
-Ultimately, images are inputted into our neural net from a 70000x3x64x64 array, and they are currently in a 70000x64x64 array. We need to add 3 channels to our images. Typically when we are working with images, the 3 channels represent the red, green, and blue components of each image. Since the MNIST dataset is grayscale, we only need 1 channel to represent our dataset. We will pad the other channels with 0's:
+Ultimately, images are inputted into the neural net from a 70000x3x64x64 array, and they are currently in a 70000x64x64 array. We need to add 3 channels to the images. Typically when we are working with images, the 3 channels represent the red, green, and blue components of each image. Since the MNIST dataset is grayscale, we only need 1 channel to represent the dataset. We will pad the other channels with 0's:
```

Comment:
I had to convince myself by reading [this](https://www.merriam-webster.com/words-at-play/is-inputted-a-word) that _inputted_ is actually a word :). 

May be paraphrasing it this way sounds better: "Ultimately, images are _**fed**_ into the neural net _**through**_ a 70000x3x64x64 array _**but**_ they are currently in a 70000x64x64 array. We need to add 3 channels to the images. ".